### PR TITLE
add prerender complete message

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes https://github.com/ampproject/amphtml/issues/2400

### CODE REVIEW

File: test/integration/test-example-validation.js [hunk 1]
```diff
@@ -74,7 +74,6 @@ describe('example', function() {
     // TODO(dvoytenko, #2600): remove this from whitelist once the message
     // contains more specific script identifier.
     /The tag \'script\' is disallowed/,
-    /\'amp-brid-player\' is disallowed/,
```
Comment (jridgewell) [inline test/integration/test-example-validation.js]: Merge conflict?

**Comment by jridgewell** [comment]
LGTM.

File: test/integration/test-example-validation.js (hunk 1 above)
Comment (erwinmombay) [inline test/integration/test-example-validation.js]: yeppp. thanks

**Pull request merged**